packages/fossil/src-main.c.patch

```diff
@@ -0,0 +1,16 @@
+--- ./src/main.c	2021-07-02 18:16:01.000000000 +0530
++++ ./src/main.c.mod	2021-08-03 07:01:36.763978266 +0530
+@@ -351,11 +351,11 @@
+   */
+   db_unsave_encryption_key();
+ #endif
+-#if defined(_WIN32) || (defined(__BIONIC__) && !defined(FOSSIL_HAVE_GETPASS))
```

Comment:
Looks like the configure script checks for termux specifically and defines FOSSIL_HAVE_GETPASS if found, but I guess the check is broken with the autosetup-cc.tcl.patch

Reply:
getpass is available in unistd.h but not freepass